docs-ref-services/eventgrid-readme-pre.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Azure Event Grid client library for JavaScript
+keywords: Azure, javascript, SDK, API, @azure/eventgrid, 
+author: maggiepint
+ms.author: magpint
+ms.date: 09/24/2020
+ms.topic: article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: javascript
+ms.service: 
+---
+
+# Azure Event Grid client library for JavaScript - Version 3.0.0-beta.2 
+
+
+[Azure Event Grid](https://azure.microsoft.com/services/event-grid/) is a cloud-based service that provides reliable event delivery at massive scale.
+
+Use the client library to:
+
+- Send events to Event Grid using either the Event Grid, Cloud Events 1.0 schemas, or a custom schema
+- Decode and process events which were delivered to an Event Grid handler
+- Generate Shared Access Signatures for Event Grid topics
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/@azure/eventgrid_3.0.0-beta.2/sdk/eventgrid/eventgrid/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid) |
+[API reference documentation](https://aka.ms/azsdk-js-eventgrid-ref-docs) |
```

Comment:
What was the idea behind using an aka.ms link here?
If I had to guess, this was supposed to point to our github.io links and once the reference docs get published to docs.microsoft.com, then that was what this would point to?

Reply:
> If I had to guess, this was supposed to point to our github.io links and once the reference docs get published to docs.microsoft.com, then that was what this would point to?

I believe that was the intention (to be completely transparent, when I started to draft the README initially I did so by taking an exiting package's README and modified it, so I sort of assumed this was the convention of the team.

When the docs are published, I'd be happy to update this link in the markdown directly to point at docs.microsoft.com.